src/app/organisation/inviteusers/post.controller.js

```diff
@@ -18,6 +18,8 @@ module.exports = (req, res) => {
     .catch((err) => {
       logger.error('Invite Users Organisation postcontroller - There was a problem inviting a user');
       logger.error(JSON.stringify(err));
-      res.render('app/organisation/inviteusers/index', { cookie, errors: err });
```

Comment:
Return the supplied fname, lname and email so the form does not lose them on validation error